app/views/bookings/index.html.erb

```diff
@@ -1,2 +1,14 @@
-<h1>Bookings#index</h1>
-<p>Find me in app/views/bookings/index.html.erb</p>
+<h1>Your bookings</h1>
+
+<div class="container">
+  <div class="cards">
+    <% @bookings.each do |booking| %>
+    <div class="booking-card">
+      <h5>Booking #<%= booking.id %></h5>
```

Comment:
> to display a reference to the user, best practice for providing unique ids for bookings

This poses a security risk when we expose ids, we can use slugs instead of direct ids from the database especially used ids